sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_amplitude_export_v1/query.sql

```diff
@@ -114,22 +117,27 @@ _previous AS (
   SELECT
     * EXCEPT (submission_date_pacific)
   FROM
-    firefox_accounts_derived.fxa_amplitude_export_v1
+    `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_amplitude_export_v1`
   WHERE
     submission_date_pacific = DATE_SUB(@submission_date, INTERVAL 1 DAY)
-    AND udf.shift_28_bits_one_day(days_seen_bits) > 0
+    AND `moz-fx-data-shared-prod.udf.shift_28_bits_one_day`(days_seen_bits) > 0
 )
 SELECT
   @submission_date AS submission_date_pacific,
   _current.* REPLACE (
     COALESCE(_current.user_id, _previous.user_id) AS user_id,
-    udf.combine_adjacent_days_28_bits(
+    `moz-fx-data-shared-prod.udf.combine_adjacent_days_28_bits`(
       _previous.days_seen_bits,
       _current.days_seen_bits
     ) AS days_seen_bits,
-    udf.combine_days_seen_maps(
+    `moz-fx-data-shared-prod.udf.combine_days_seen_maps`(
       _previous.os_used_month,
-      ARRAY(SELECT STRUCT(key, CAST(TRUE AS INT64) AS value) FROM _current.os_used_month AS key)
+      ARRAY(
+        SELECT
+          STRUCT(key, CAST(TRUE AS INT64) AS value)
+        FROM
+          `moz-fx-data-shared-prod._current.os_used_month` AS key
```

Comment:
Not sure what's happening here. Looks like `_current` is an alias for a CTE but it's being qualified

Reply:
I found a way to extract table aliases and ignore references related to these aliases, similar to what was done with CTEs already